python/rikai/spark/sql/codegen/pytorch.py

```diff
@@ -74,7 +75,9 @@ def torch_inference_udf(
                     if isinstance(batch, torch.Tensor):
                         batch = batch.to(device)
                     predictions = model(batch)
-                    if spec.post_processing:
+                    if _number_of_params(spec.post_processing) == 2:
+                        predictions = spec.post_processing(batch, predictions)
```

Comment:
do we want to have these optional parameters that we try to figure out whether it's present? I'm wondering whether it would be better to be more explicit here? Options:
1. Just have predictions and model be required arguments
2. declare the post processing interface in the model config
3. something else?

Reply:
> Just have predictions and model be required arguments

Typo? Just have predictions and batch (model input) be required arguments?

Reply:
> Just have predictions and model be required arguments

Cool idea